client/src/components/TeamManagement/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { inviteAdminUser, useAdminUsers } from '@enterprise-oss/osso';
+import { Button, Card, Form, Input, Modal, Select, Table } from 'antd';
+import React, { ReactElement, useState } from 'react';
+
+import styles from './index.module.css';
+
+export default function TeamManagement(): ReactElement {
+  const { data, loading } = useAdminUsers();
+  const { inviteUser } = inviteAdminUser();
+  const [modalOpen, setModalOpen] = useState(false);
+  const [form] = Form.useForm();
+
+  return (
+    <Card>
+      <div className={styles.cardTopRow}>
+        <h2>Team</h2>
+        <Button size="small" onClick={() => setModalOpen(true)}>
+          Invite Teammate
+        </Button>
+      </div>
+      <Table
+        loading={loading}
+        rowKey="id"
+        dataSource={data?.adminUsers}
+        className={styles.table}
+      >
+        <Table.Column title="Email" dataIndex="email" key="email" />
+        <Table.Column
+          title="Last login"
+          dataIndex="updatedAt"
+          key="updatedAt"
+        />
+        <Table.Column title="Role" dataIndex="role" key="role" />
+        <Table.Column title="Actions" dataIndex="domain" key="domain" />
```

Comment:
Is it possible to hide "Actions" if the user's not a super admin? If it's blank it feels like a waste of space.

Reply:
its always blank rn, lets just remove it and circle back. we dont yet have any api support for actions (remove, change role?)

also, non admins wont be able to access this screen at all

Reply:
cool makes sense